ocaml/database/db_xml.ml

```diff
@@ -12,6 +12,7 @@
  * GNU Lesser General Public License for more details.
  *)
 open Db_cache_types
+open Xapi_stdext_unix
```

Comment:
Please move this closer to where its used, i.e. `Xapi_stdext_unix.Unixext.fsync` if it is used in only one place.